GOVERNANCE.md

```diff
@@ -13,48 +13,39 @@ You can contact the project maintainers at any time by sending an email to the
 
 Main responsibilities of maintainers include:
 
-1) They share responsibility in the project's success.
-2) They have made a long-term, recurring time investment to improve the project.
-3) They spend that time doing whatever needs to be done, not necessarily what
-is the most interesting or fun.
+1) Sharing responsibility for the project's success.
+2) Making a long-term, recurring time investment to improve the project.
+3) Performing necessary tasks, even if they are not the most interesting or fun.
 
 ## Reviewers
 
-A reviewer is a core role within the project.
-They share in reviewing issues and pull requests. Their pull request approvals
-are needed to merge a large code change into the project.
+A reviewer is a core role within the project. They share in reviewing issues and pull requests.
+Their pull request approvals are needed to merge significant code changes into the project.
+
+## Emeritus Maintainers
+
+Emeritus maintainers are retired maintainers who have chosen to remain involved in the project as advisors.
```

Comment:
I wonder if it would be good to soften it a bit to include folks who can't remain involved (even as advisors) just as a way to say "[thanks for all the fish](https://ell.stackexchange.com/questions/115782/what-does-the-line-so-long-and-thanks-for-all-the-fish-mean)".

Perhaps:
Emeritus maintainers are retired maintainers who have provided valuable contributions to the project in the past but are not able to dedicate the time necessary to be fully active maintainers going forward. While their efforts will be focused elsewhere, it is hoped that they will try to find the time to continue to be active participants in the community by:
1) providing....


or something like that